magma/magma
# chore(redirectd): redirectd is bazelified

Signed-off-by: Lars Kreutzer <lars.kreutzer@tngtech.com>



## Summary

- Module redirectd is bazelified including the test. 
- Minor refactoring of mobilityd dependencies -> move dependencies to where they are actually needed.

## Test Plan

-  Run redirectd: 
   - `bazel run //lte/gateway/python/magma/redirectd:redirectd`
   - Note 1: This is expected to fail in the dev container, because needed services are not running.
   - Note 2: Runs in the magma-VM
- Run test: 
  - `bazel test //lte/gateway/python/magma/redirectd/tests:all`

## Additional Information
